sdk/test/ProtocolTests/AWSSDK.ProtocolTests.Net45.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+     <RunAnalyzersDuringBuild Condition="'$(RunAnalyzersDuringBuild)'==''">true</RunAnalyzersDuringBuild>
+      <TargetFramework>net45</TargetFramework>
+      <DefineConstants>$(DefineConstants);TRACE;BCL;BCL45;ASYNC_AWAIT;DEBUG</DefineConstants>
+      <DebugType>portable</DebugType>
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <AssemblyName>AWSSDK.ProtocolTests.Net45</AssemblyName>
+      <PackageId>AWSSDK.ProtocolTests.Net45</PackageId>
+  
+      <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+      <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+      <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+      <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+      <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+      <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+      <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+      <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+      <SignAssembly>true</SignAssembly>
```

Comment:
Also, do we need to include `<SignAssembly>true</SignAssembly>` if we're not providing the `<AssemblyOriginatorKeyFile>`? What I'm trying to get at is that I don't think we're actually signing this assembly since we don't provide it.  When I tried to include the <Choose> statement that we usually add in here it was causing issues since some of its dependencies aren't strong-named signed as well. I think we can remove this, but not 100% sure.